### ISSUE TITLE
Panic on missing Curves in tests/examples/missing_curves_section.las

### ISSUE THREAD

**Issue by dcslagel** [MandarJKulkarni/lascheck#2]
When running the following standard example lascheck panics on missing curve:

Script:
```python
import lascheck
myfile = 'tests/examples/missing_curves_section.las'
las = lascheck.read(myfile)
print('-------------------------------------------')
print(myfile, las.check_conformity())
print('-------------------------------------------')
print(las.get_non_conformities())
```

Output:
```python
Header section Curves regexp=~C was not found.
-------------------------------------------
tests/examples/missing_curves_section.las False
-------------------------------------------
Traceback (most recent call last):
  File "runner.py", line 26, in <module>
    print(las.get_non_conformities())
  File "/usr/local/devel/MyWrk/py-wrk/lascheck/lascheck/lascheck/las.py", line 701, in get_non_conformities
    if (spec.ValidIndexMnemonic.check(self)) is False:
  File "/usr/local/devel/MyWrk/py-wrk/lascheck/lascheck/lascheck/spec.py", line 91, in check
    if las_file.curves[0].mnemonic == "DEPT" or \
  File "/usr/local/devel/MyWrk/py-wrk/lascheck/lascheck/lascheck/las.py", line 428, in curves
    return self.sections["Curves"]
KeyError: 'Curves'
```

Earlier LasCheck ran `spec.MandatorySections.check(self))`.  Should its result be a dependency for all later tests for any of the mandatory sections?

DC

**Comment by MandarJKulkarni** [comment]
This is a major issue and needs major change. Thanks for raising this.

**Comment by dcslagel** [comment]
This looks to be fixed now on master with the commit 130e5957.

Here is my current test run.  It does not have the panic and reports the check result properly.
```python
Header section Curves regexp=~C was not found.
-------------------------------------------
tests/examples/missing_curves_section.las False
-------------------------------------------
['Missing a mandatory section']
```

Closing this issue.

Thank you,
DC
